frontend/components/App/App.tsx

```diff
@@ -65,6 +65,7 @@ const App = ({ children }: IAppProps): JSX.Element => {
   useDeepEffect(() => {
     // on page refresh
     if (!user && authToken()) {
```

Comment:
Maybe `await authToken()` here?

Reply:
@RachelElysia would you give the above a try?

Reply:
nvm I don't know what I was thinking there :/

Reply:
"'await' expressions are only allowed within async functions and at the top levels of modules"

Reply:
@RachelElysia, I think the async piece I was looking for is the redux action dispatch I mentioned in the LogoutPage component and I was thinking the async action could be causing the race. So we can try @martavis' idea there and/or look into including some sort of `await` or `then`/`finally` to add some controls around then the redirect happens and make sure to wait for it to clear the auth token